storage/local/interface.go

```diff
@@ -26,6 +26,8 @@ import (
 // are goroutine-safe. Storage implements storage.SampleAppender.
 type Storage interface {
 	prometheus.Collector
+	Querier
```

Comment:
I was thinking both of the same things :)

I initially left the `Append()` in here because it had a more specific docstring about the local storage than the generic Appender interface. But I could also just add the extra comments above the embedded interface.

Collector could be moved out, but then we need to make another type that combines everything again.